src/Illuminate/Container/Container.php

```diff
@@ -285,10 +286,16 @@ public function bindMethod($method, $callback)
      *
      * @param  array|string $method
      * @return string
+     *
+     * @throws \InvalidArgumentException
      */
     protected function parseBindMethod($method)
     {
         if (is_array($method)) {
```

Comment:
I'd use early returns here to avoid the awkward nested `if`s.

Reply:
I had idea to write like 
```PHP
    protected function parseBindMethod($method)
    {
        if (is_array($method) && ($count = count($method) < 2)) {
            throw new InvalidArgumentException("Method should be array with length >= 2, array with length [{$count}] given");
        }
    
        if (is_array($method)) {
            return $method[0] . '@' . $method[1];
        }

        return $method;
    }
```

But I`m not sure... is it really better?